Gruntfile.js

```diff
@@ -24,18 +24,25 @@ module.exports = function(grunt) {
 			}
 		},
 		"saucelabs": {
+			debug: {
+				options: {
+					urls: { all: 'http://127.0.0.1:3000/test/director?mode=all' },
```

Comment:
I think this should grab port from the process.env if it exists as that is what the service will use.